website/docs/workflow-analytics.md

```diff
@@ -0,0 +1,67 @@
+---
+id: workflow-analytics
+title: Workflow Analytics Using Litmus-Portal
+sidebar_label: Workflow Analytics
+---
+
+---
+
+Once users have successfully scheduled a workflow (Recurring/ Non-recurring), The users can check how there workflows are running, analyse the workflow runs Hourly/daily/Monthly using workflow dashboards and check how their system behaves when chaos injection happens using Application dashboards.
+
+### Overview
+
+Overview Tab helps users to manage their connected Datasources and Dashboards & makes it easier to view analyse the workflows. Overview Tab is divided into 3 parts -
+
+- Data Sources
+- Dashboards
+- Workflow Dashboards
+
+> When no agent is connected or in active state, the overwiew page will show a banner for connecting an agent. Users can deploy a new agent using [litmusctl](https://github.com/litmuschaos/litmusctl).
+
+<img src={require('./assets/overview-agent-connect.png').default} width="1000" height="500" />
+
+Once the agent is connected, Users will see a banner for adding datasources and list of recently updated workflow runs (if any).
+
+<img src={require('./assets/overview-first-look.png').default} width="1000" height="500" />
+
+For Adding a Data source, Users can follow here
+
+Once users have added a data source sucessfully, they will be able to see the data source listed on the overview page.
+
+<img src={require('./assets/overview-with-datasource.png').default} width="1000" />
+
+Now, users will get an banner with an option to add a dashboard for visualizing the data from the connected the data sources.
+
+For Adding a Dashboard, Users can follow here, After setting up the dashboard, the users will be able to see the the dashboard on overview page.
+
+<img src={require('./assets/overview-with-dashboard.png').default} width="1000" />
+
+Users can check workflow analytics and workflow run graph by clicking on _**see workflow run**_ & _**see workflow analytics**_ icons for a workflow from list of workflows shown on overview page.
+
+### Workflow Analytics
+
+As we have already discussed about the workflow run graph in other parts of documentation, we can proceed with workflow analytics. On clicking _**see workflow analytics**_, Users will be redirected to workflow analytics page for that particular workflow.
+
+<img src={require('./assets/workflow-analytics.png').default} width="1000" height="500" />
+
+On workflow analytics page, In the first container, user will be able to see the details about the selected workflow like Worklflow name, workflow id, agent details like name and namespace and workflow run details like last run and it's Regularity.
```

Comment:
- `In` -> `in`
- `name and namespace and workflow run details` -> `name, namespace, and workflow run details`
- `it's` -> `its`